packages/components/src/components/text/text.jsx

```diff
@@ -1,27 +1,27 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
+const Text = ({ children, size, color, align, weight, line_height, as, className, ...props }) => {
     function setStyle() {
```

Comment:
Isn't it better to write an arrow function instead?

Reply:
const Text = ()=>{....}
Don't you mean this?

Reply:
yes, exactly